src/Headers.ts

```diff
@@ -1,3 +1,4 @@
+import { splitCookiesString } from 'set-cookie-parser'
```

Comment:
Hey, @mattcosta7 👋 

I'm adding a set-cookie-parser to correctly parse the `Set-Cookie` response header since it may contain commas that are not header value delimiters (like commas in the `Expires` portion of the cookie header). 

This resolves #60. 

Could you please take a look at this change just to make sure it's alright? Thank you.